src/amberscript/parser.cc

```diff
@@ -1807,7 +1807,7 @@ Result Parser::ParseRun() {
 
     auto cmd = MakeUnique<DrawGridCommand>(pipeline);
     cmd->SetLine(line);
-    cmd->SetPolygonMode(pipeline->GetPolygonMode());
```

Comment:
The other two no-longer set polygon mode, why does this one? Seems inconsistent. I think the DrawGridCommand should be changed to accept the pipeline and SetPolygonMode removed from cmd.

Reply:
I agree. Let me try that.